scenes/e2e/src/app.e2e-spec.ts

```diff
@@ -0,0 +1,18 @@
+import {AppPage} from './app.po';
+import {screenshot} from '../screenshot';
+
+describe('screenshot scenes for each component', () => {
+  let page: AppPage;
+  const components = ['button', 'checkbox', 'datepicker', 'input'];
+
+  beforeEach(() => {
+    page = new AppPage();
+  });
+
```

Comment:
Can you just add a comment about how we're not really testing anything and just using this as a convenient way to take snapshots of different pages